calcom/cal.com
# Close settings sidebar on window resize

## What does this PR do?



This PR adds an event listener to close the settings sidebar when the window reaches the "large" (1024px) size.

Fixes #4642


https://www.loom.com/share/36f07579cf9a4bcfbac5d5c66aef900a

**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Go to the settings page and reduce the window size to where the top bar renders
- Open the side navigation pane
- Resize the window back to large
	- There should only be one sidebar container

## Checklist


- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
